tests/test_responses.py

```diff
@@ -327,7 +327,7 @@ def test_head_method(test_client_factory):
 
 
 def test_empty_response(test_client_factory):
-    app = Response()
+    app = Response(status_code=200)
     client: TestClient = test_client_factory(app)
     response = client.get("/")
     assert response.headers["content-length"] == "0"
```

Comment:
I'd expect to see this behaviour...

```suggestion
    app = Response()
    client: TestClient = test_client_factory(app)
    response = client.get("/")
    assert response.headers["content-length"] == "0"
    assert response.status_code == 204
```